$OpenBSD: patch-lib_tmuxinator_config_rb,v 1.6 2019/09/01 11:10:36 jasper Exp $

Prevent: "tmux: unknown option -- V" and just report the latest version
for the value is used to decide if it's newer than 2.2

Index: lib/tmuxinator/config.rb
--- lib/tmuxinator/config.rb.orig
+++ lib/tmuxinator/config.rb
@@ -61,15 +61,7 @@ module Tmuxinator
       end
 
       def version
-        if Tmuxinator::Doctor.installed?
-          tmux_version = `tmux -V`.split(" ")[1]
-
-          if tmux_version == "master"
-            TMUX_MASTER_VERSION
-          else
-            tmux_version.to_f
-          end
-        end
+        2.3
       end
 
       def default_path_option
